package-lock.json

```diff
@@ -5,12 +5,11 @@
   "packages": {
     "": {
       "name": "tldr-pages",
-      "hasInstallScript": true,
```

Comment:
Out of curiosity, why did this get removed?

Reply:
Yes. npm 7 saves some meta-data about the run-scripts in the package.json file within the lock file. When I moved husky from postinstall to prepare in #5711, that altered the package-lock.json file for this line technically, but since I manually editing the file through GH, that change was not propagated.